src/utils/numeral.ts

```diff
@@ -1,7 +1,14 @@
 import numeral from 'numeral';
 import 'numeral/locales'; // Without this numeral will throw an error when using not 'en' locale
 
-import {i18n} from './i18n';
+import {Lang, i18n} from './i18n';
+
+// Set space delimeter for all locales possible in project
+Object.values(Lang).forEach((value) => {
+    if (numeral.locales[value]) {
+        numeral.locales[value].delimiters.thousands = ' ';
+    }
+});
```

Comment:
You configured numeral almost correctly, but it seems you deleted "," in format string. In format "," is delimiter sign, not delimiter itself, so no delimiter was applied.

I made more simple delimiter override, so it should work and no so complex for future support